files/ubuntu-18.04/sge_preinstall.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+set -e
+echo "Downloading and extracting source packages for $dest_package"
+
+orig_package='gridengine_8.1.9+dfsg'
+deb_package='gridengine_8.1.9+dfsg-9build1'
+dest_package="$TARBALL_ROOT_DIR"
+package_url='https://bugs.launchpad.net/ubuntu/+archive/primary/+sourcefiles/gridengine/8.1.9+dfsg-9build1'
+
+wget $package_url/$orig_package.orig.tar.gz
+wget $package_url/$deb_package.debian.tar.xz
+wget $package_url/$deb_package.dsc
```

Comment:
is there any dpkg or apt command that would avoid to download three packages directly? e.g. apt source

Reply:
Solved by using apt source